docs/processing_information.md

```diff
@@ -0,0 +1,64 @@
+Recent publications highlight that a small fraction of reads from single-cell experiments are derived from intronic or intergenic sequences and not from spliced cDNA [Kaminow _et al._ 2021](https://www.biorxiv.org/content/10.1101/2021.05.05.442755v1.full#sec-5).
+Alignment to the transcriptome alone, without inclusion of intronic regions, can result in an inaccurate alignment of the intronic sequences to spliced cDNA that share sequence similarity with a spliced transcript. 
+This has been observed to lead to an increase in spuriously detected genes [He _et al._ 2021](https://www.biorxiv.org/content/10.1101/2021.06.29.450377v1.full.pdf).
+In our hands, we have found that use of the `splici` index led to a more comparable distribution of unique genes found per cell to Cell Ranger than use of an index obtained from spliced cDNA transcripts. 
+
+#### Selective Alignment
+
+Alevin-fry utilizes the selective alignment strategy invoked by `salmon` to align reads. 
+Briefly, selective alignment uses a mapping score validated approach to identify maximal exact matches between reads and the provided index. 
+For all samples, we used selective alignment to the `splici` index. 
+
+A more detailed description of the alignment strategy invoked by `salmon` in conjuction with alevin-fry can be found [here](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02151-8). 
+
+#### Alevin-Fry Parameters 
+
+After we aligned FASTQ files using selective alignment to the `splici` index, we continued with the alevin-fry pipeline using the following parameters: 
+
+1. During the [`generate-permit-list` step of Alevin-fry](https://alevin-fry.readthedocs.io/en/latest/generate_permit_list.html), we utilized the `--unfiltered-pl` option, which returns any cell with at least 1 read found in the reference barcode list that is used. 
+For our reference barcode list, we used a list of all possible cell barcodes from 10X Genomics.
+Use of this option allowed us to obtain an unfiltered counts matrix with all possible cell barcodes detected in the experiment. 
+
+2. Alevin-fry provides multiple options for performing [feature quantification and UMI de-duplication](https://alevin-fry.readthedocs.io/en/latest/quant.html). 
+We chose to use the `cr-like-em` resolution strategy for UMI de-duplication. 
+The `cr-like-em` resolution strategy assigns all UMI's that align to a single gene to that gene. 
+For any gene that is assigned to more than one gene, the UMI is assigned to the gene with the highest count. 
+
+### Post Alevin-Fry Processing
+
+#### Combining counts from spliced cDNA and intronic regions
+
+For single-cell samples, we only included reads aligning to spliced cDNA transcripts in the counts matrix. 
+For single-nuclei samples, all counts for spliced cDNA and intronic regions were summed for each gene to return the total counts summarized by gene in the counts matrix. 
+
+#### Filtering Cells
+
+### Samples with CITE-seq
```

Comment:
Similar to a prior suggestion, making this less about samples and more about the processing. I anticipate the first sentence will be something like "For samples with CITE-seq data, we..."
```suggestion
### CITE-seq quantification
```